com.github.bajoja.indicator-kdeconnect.json

```diff
@@ -0,0 +1,186 @@
+      "buildsystem": "simple",
+      "build-commands": [
+        "python3 setup.py install --prefix=/app --root=/"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://pypi.python.org/packages/fa/2e/25f25e6c69d97cf921f0a8f7d520e0ef336dd3deca0142c0b634b0236a90/oauthlib-2.0.2.tar.gz#md5=f59b448a30684347e00f369bc3542abc",
+          "sha256": "b3b9b47f2a263fe249b5b48c4e25a5bce882ff20a0ac34d553ce43cff55b53ac"
+        }
+      ]
+    },
+    {
+      "name": "python-request_oauthlib",
+      "buildsystem": "simple",
+      "build-commands": [
+        "pip3 install --prefix=/app requests_oauthlib-0.8.0-py2.py3-none-any.whl"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://pypi.python.org/packages/77/34/d0957563f20b259a31c12f14e858d79f2e66eb539d3c1b9ab7077ef030ca/requests_oauthlib-0.8.0-py2.py3-none-any.whl#md5=3dd6a56cde1e4b86cbdec252b95efd8c",
+          "sha256": "50a8ae2ce8273e384895972b56193c7409601a66d4975774c60c2aed869639ca"
+        }
+      ]
+    },
+    {
+      "name": "indicator-kdeconnect",
+      "buildsystem": "cmake",
+      "build-options": {
+        "env": {
+          "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+        }
+      },
+    "post-install": [
+      "cp data/desktop/indicator-kdeconnect-settings.desktop /app/share/applications/com.github.bajoja.indicator-kdeconnect.settings.desktop",
+      "rm /app/share/applications/indicator-kdeconnect-settings.desktop"
+    ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://www.github.com/Bajoja/indicator-kdeconnect.git",
+          "branch": "0.9.2",
+          "commit": "86add058ee2307adacc6adfd2b3bed3fb041cebd"
+        },
+        {
+          "type": "patch",
+          "path": "indicatorkdeconnect-fixes.patch"
+        }
+      ]
+    },
+    {
+      "name": "icons",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -dm755 /app/share/icons/hicolor",
+        "install -dm755 /app/share/icons/Adwaita",
+        "install -dm755 /app/share/icons/elementary",
+        "install -dm755 /app/share/icons/ubuntu-mono-light",
+        "install -dm755 /app/share/icons/ubuntu-mono-dark",
+        "tar -xf icons.tar.gz",
+        "cp -aL hicolor/* /app/share/icons/hicolor",
+        "cp -aL Adwaita/* /app/share/icons/Adwaita",
+        "cp -aL elementary/* /app/share/icons/elementary",
+        "cp -aL ubuntu-mono-light/* /app/share/icons/ubuntu-mono-light",
+        "cp -aL ubuntu-mono-dark/* /app/share/icons/ubuntu-mono-dark"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "icons.tar.gz"
+        }
+      ]
+    },
+    {
+      "name": "appdata",
+      "buildsystem": "simple",
+      "build-commands": [
+          "install -Dm644 com.github.bajoja.indicator-kdeconnect.appdata.xml app/share/appdata/com.github.bajoja.indicator-kdeconnect.appdata.xml"
```

Comment:
I think you're going to need an extra / here e.g.` /app/share/appdata/com.github.bajoja.indicator-kdeconnect.appdata.xml`